lib/python/rose/reporter.py

```diff
@@ -78,6 +79,40 @@ def __init__(self, verbosity=DEFAULT, contexts=None, raise_on_exc=False):
         self.event_handler = None
         self.raise_on_exc = raise_on_exc
 
+    def format_msg(self, msg, verbosity, prefix=None, clip=None):
+        """Format a message for reporting."""
+
+        msg_lines = []
+
+        if verbosity >= self.VV:
+            stamp = time.strftime("%Y-%m-%dT%H:%M:%SZ : ")
```

Comment:
(I think a single space instead of space-colon-space is sufficient as a delimiter.)